.github/workflows/crucible-scheduled.yaml

```diff
@@ -0,0 +1,17 @@
+name: crucible-scheduled
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
```

Comment:
workflow_dispatch allows manual trigger, and it will use the default settings I assume

Reply:
What do you mean by "default settings"?  If you manually trigger this workflow (through the web gui for example) it will run just like if it were scheduled to run at that time, the parameters to the workflow will be exactly the same as if it were running via the cron job like interface.